recipes/qcodes/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "qcodes" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "a70f3a209e2f4371634b12094def37696fa2accb2f0746e5dffba50e4918e33e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://github.com/QCoDeS/Qcodes/archive/v{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py<35]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - numpy >=1.10
+    - pyvisa >=1.8
+    - ipython >=4.1.0
+    - jupyter >=1.0.0
+    - ipywidgets >=4.1
+    - h5py >=2.6
+    - pathlib
+    - matplotlib
+    - coverage
+    - pyqtgraph
+
+test:
+#  source_files:
+#    - qcodes/test.py
+#  commands:
+#    - python qcodes/test.py
```

Comment:
Should be possible to uncomment this in the feedstock as we now use `conda-build` 2.